client/src/App.js

```diff
@@ -1,53 +1,85 @@
 import React from "react";
-import { BrowserRouter, Route,
+import {
+  BrowserRouter, Route,
   Redirect,
-  Switch } from "react-router-dom";
+  Switch
+} from "react-router-dom";
 
 import SigninPage from "./user/pages/SignInPage";
 import SignUpPage from "./user/pages/SignUpPage";
 import { AuthContext } from './shared/context/auth-context';
 import { useAuth } from './shared/hooks/auth-hook';
+
 import Box from "@material-ui/core/Box";
 import CssBaseline from "@material-ui/core/CssBaseline";
-import { Header } from './components/Header';
-import { Dashboard } from './pages/Dashboard';
-
-const  App = () => {
-  const { token, login, logout, userId } = useAuth();
-
-  let routes;
-
-  routes = (
-    <Switch>
-      <Route path="/" exact>
-        <SigninPage />
-      </Route>
-     
-      <Route path="/signup">
-        <SignUpPage />
-      </Route>
-      
-      <Redirect to="/" />
-    </Switch>
-  );
-
-  return (
-    <AuthContext.Provider
-      value={{
-        isLoggedIn: !!token,
-        token: token,
-        userId: userId,
-        login: login,
-        logout: logout
-      }}>
-  
-   <BrowserRouter>
-     {routes}
-   </BrowserRouter>
-   
-   </AuthContext.Provider>
-  );
 
+import { Home } from './pages/Home';
+
+import { ThemeProvider } from '@material-ui/core/styles';
+import { theme } from './themes/theme';
+
+
+// Varun's work start
```

Comment:
I don't really understand why this is commented out - since Varun's structure was merged into dev ahead of time, we should be follow this structure. What he added was 
- Routes 
- Authentication

in addition, this is follow a functional component which the below work is class based

Reply:
Varun's codes lead to the `login` page while my work lead to the `dashboard`. We need both entry points to do the development, at least I need, before we finish the redirection work.

Reply:
is there a reason why we can't merge both of these works together? 

we should definitely not merge this commented out code into dev